forte/pipeline.py

```diff
@@ -466,6 +467,19 @@ def save(self, path: str):
         with open(path, "w", encoding="utf-8") as f:
             yaml.safe_dump(self._dump_to_config(), f)
 
+    def export(self, name: str):
```

Comment:
I remember we discussed a method to generate a name when none is given (and use auto increment numbers for new files)

Reply:
Anything can be a user's job here, the discussion and spirit behind all this would be to make sure care as little as possible.

Now, we allow someone to use a name for sure, but it can be even simpler if we just use a default name, and the user don't need to worry about it if he is not planning to use multiple pipeline submission.

Uniqueness check can be another thing, one approach is we always overwrite the previous one if the name is not unique, that's reasonable too.